### DISCUSSION TITLE
Getting odisAuthError while calling OdisUtils.Quota.getPnpQuotaStatus()

### DISCUSSION THREAD

**Discussion by viral-sangani** [celo-org/social-connect#210] [Help (Q&A)]
I was trying to using ODIS in alfajores, but keep getting `odisAuthError` for almost all the calls made using `OdisUtils`.

I have checked for cUSD balance and enough allowance. 
> Note: Calls made directly to ODIS contract are working. I was able to call `payInCUSD` directly.

Error -
```
⚠  External network resource requested!
   - URL: "https://us-central1-celo-phone-number-privacy.cloudfunctions.net/combiner/quotaStatus"
 - Be careful, this may be a production service.
>  Error: odisAuthError
```

Env - Firebase Functions (Nodejs 18).

**Comment by therealharpaljadeja** [comment]
Make sure the DEK (public key) set in `Accounts.sol` corresponds to the private DEK you are using in your code.
